src/main/resources/application-custom.yaml

```diff
@@ -143,6 +146,9 @@ hapi:
       revocation_url: ${OAUTH_REVOCATION_URL}
       manage_url: ${OAUTH_MANAGE_URL}
 
+    basic_auth:
+      enabled: ${BASIC_AUTH_ENABLED:false}
```

Comment:
I think to make it work with basic auth, we will need to have username and password here as well.

Reply:
I was still testing and hadn't assigned any reviewers, but good catch. I'll also add something to my local docker-compose to ensure that I test with the custom yaml file.